lib/spack/spack/test/database.py

```diff
@@ -524,7 +524,7 @@ def read_and_modify():
         with mutable_database.write_transaction():
             _mock_remove('mpileaks ^zmpi')
 
-    p = multiprocessing.Process(target=read_and_modify, args=())
+    p = fork_context.Process(target=read_and_modify, args=())
```

Comment:
Generally I think we want to stick with `multiprocessing.Process` vs. `fork_context` so long as it doesn't cause us a problem.

Reply:
This one is actually required:
```
AttributeError: Can't pickle local object 'test_030_db_sanity_from_another_process.<locals>.read_and_modify'
```